code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -352,9 +359,26 @@
 	R.my_atom = src.emag
 	R.add_reagent("beer2", 50)
 	src.emag.name = "Mickey Finn's Special Brew"
-	return
-
 
+/obj/item/weapon/robot_module/butler/recharge_consumable(var/mob/living/silicon/robot/R)
```

Comment:
Found it, it's `restock()`

https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/game/machinery/rechargestation.dm#L259